BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/de.lproj/Localizable.strings

```diff
@@ -84,3 +84,16 @@
 
 "ginibank.digitalinvoice.edit.name.error" = "Bitte geben Sie den Artikelnamen ein.";
 "ginibank.digitalinvoice.edit.price.error" = "Bitte geben Sie den Artikelpreis ein.";
+
+"ginibank.skonto.screen.title" = "Skonto";
+"ginibank.skonto.backButton.title" = "Zurück";
+"ginibank.skonto.info.title" = "Mit Skonto";
+"ginibank.skonto.info.status" = "• Aktiviert";
+"ginibank.skonto.info.message" = "Zahlung in %@: %@ Skonto.";
```

Comment:
[clarification] we need to have also Tagen/days in this string. Please add a TODO if you can not do it now.

Reply:
I will leave TODO here in terms I don't know how it will be specified. I saw in backend response example time `skontoDuePeriod` "2 weeks" and thought it will be not always days

Reply:
We will not use `skontoDuePeriod` to display this message. We will use `remainingDays` which will be in days.
```
"skontoRemainingDays":{
               "entity":"timeperiod",
               "value":"2:day"
            }
```

Reply:
Also if is possible to be fixed now, let's fix it now and not let it for later. Thank you!

Reply:
Thanks for clarification! In that case its really a good idea to include "days" in localization string